paper-icon-button-light.html

```diff
@@ -0,0 +1,82 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../paper-behaviors/paper-ripple-behavior.html">
+
+<!--
+@group Paper Elements
+@element paper-icon-button-light
+@demo demo/paper-icon-button-light.html
+-->
+<dom-module id="paper-icon-button-light">
+  <template strip-whitespace>
+    <style>
+      :host {
+        vertical-align: middle;
+        color: inherit;
+        outline: none;
+        width: 24px;
+        height: 24px;
+        background: none;
+        margin: 0;
+        border: none;
+        padding: 0;
+
+        position: relative;
+        cursor: pointer;
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+      }
+
+      :host([disabled]) {
+        color: #9b9b9b;
```

Comment:
For now, I'm going to leave this hard-coded out of my (untested) suspicion that the CSS variables shim will make this slower. (Could be marginal on top of the CSS shim cost in general?) I mentioned this to @cdata a moment ago and he made the point that we can always change this later if we determine that it's not expensive.